src/type_conversion/testing.h

```diff
@@ -0,0 +1,90 @@
+#ifndef TEST_H
+#define TEST_H
+#ifdef TEST
+
+#include "Python.h"
+#include <stdio.h>
+
+int
+testing_init();
+
+/**
+ *
+ * We define two macros to make tests:
+ * DEFINE_TEST(name, body):
+ *   Defines a test with given name and body. Test should use ASSERT macro
+ *   above.
+ *
+ * DEFINE_TEST_EXPECT_FAIL(name, match, body):
+ *    This variant expects the test to fail with a message that matches js regex
+ *    "match".
```

Comment:
Could you add a section to the contributing documentation with these two macros?

Reply:
I will put it in #1048.